.env.example

```
@@ -0,0 +1,48 @@
+
+REACT_APP_FALLBACK_LANGUAGE=en
+REACT_APP_MAIN_LANGUAGE=uk
+
+REACT_APP_API_URL=https://yourserver.io/api/
+
+# TODO: change API URL to correct one before deploying:
+REACT_APP_API_BASE_URL=http://localhost:3000/api
+REACT_APP_CONTINUOUS_DEPLOYMENT_HEADER_NAME=no-aws-header-name
+REACT_APP_CONTINUOUS_DEPLOYMENT_HEADER_VALUE=no-aws-header-value
+REACT_APP_MOCKOON_URL=http://${WEBSITE_DOMAIN}:${MOCKOON_PORT}
+REACT_APP_GRAPHQL_URL=http://${WEBSITE_DOMAIN}:${GRAPHQL_PORT}/${GRAPHQL_API_PATH}
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REACT_APP_GRAPHQL_URL key should go before the REACT_APP_MOCKOON_URL key

(UnorderedKey)